inc/class-wp-saml-auth-cli.php

```diff
@@ -0,0 +1,117 @@
+<?php
```

Comment:
help text/description is missing when running `wp`.


```
NAME

  wp

DESCRIPTION

  Manage WordPress through the command-line.

SYNOPSIS

  wp <command>

SUBCOMMANDS

  cache               Manage the object cache.
  cap                 Manage user capabilities.
  cli                 Get information about WP-CLI itself.
  comment             Manage comments.
  core                Download, install, update and otherwise manage WordPress proper.
  cron                Manage WP-Cron events and schedules.
  db                  Perform basic database operations.
  eval                Execute arbitrary PHP code.
  eval-file           Load and execute a PHP file.
  export              Export content to a WXR file.
  help                Get help on WP-CLI, or on a specific command.
  hook                List callbacks registered to a given action or filter.
  import              Import content from a WXR file.
  launchcheck         Implements example command.
  media               Manage attachments.
  menu                List, create, assign, and delete menus
  network             
  option              Manage options.
  package             Manage WP-CLI packages.
  plugin              Manage plugins.
  post                Manage posts.
  post-type           Manage post types.
  rewrite             Manage rewrite rules.
  role                Manage user roles.
  saml-auth           
  scaffold            Generate code for post types, taxonomies, etc.
  search-replace      Search/replace strings in the database.
  server              Launch PHP's built-in web server for this specific WordPress installation.
  shell               Interactive PHP console.
  sidebar             Manage sidebars.
  site                Perform site-wide operations.
  super-admin         List, add, and remove super admins from a network.
  taxonomy            Manage taxonomies.
  term                Manage terms.
  theme               Manage themes.
  transient           Manage transients.
  user                Manage users.
  widget              Manage sidebar widgets.

```